html/assets/machinery.js

```diff
@@ -30,7 +30,9 @@ $(document).ready(function () {
   $.each(scopes, function(index, scope) {
     templates[scope] = Hogan.compile($("#scope_" + scope).html());
   });
-  template = Hogan.compile($('#content').html());
+  templates["inspection_details_template"] =
```

Comment:
['inspection_details_template'] is better written in dot notation.<br>'templates' is not defined.